packages/chevrotain/src/parse/parser/traits/gast_recorder.ts

```diff
@@ -172,25 +172,25 @@ export class GastRecorder {
         delete (this as any)[`AT_LEAST_ONE_SEP${idx}`]
       }
 
-      delete this[`consume`]
-      delete this[`subrule`]
-      delete this[`option`]
-      delete this[`or`]
-      delete this[`many`]
-      delete this[`atLeastOne`]
-
-      delete this.ACTION
-      delete this.BACKTRACK
-      delete this.LA
+      delete (this as any)[`consume`]
+      delete (this as any)[`subrule`]
+      delete (this as any)[`option`]
+      delete (this as any)[`or`]
+      delete (this as any)[`many`]
+      delete (this as any)[`atLeastOne`]
+
+      delete (this as any).ACTION
+      delete (this as any).BACKTRACK
+      delete (this as any).LA
     })
   }
 
   // TODO: is there any way to use this method to check no
   //   Parser methods are called inside an ACTION?
   //   Maybe try/catch/finally on ACTIONS while disabling the recorders state changes?
+  // @ts-ignore
```

Comment:
1. use `@ts-expect-error`
2. add a comment `-- blah blah` why 

There is an ESLint rule for this, but I have not been able to activate it due to old code.

Reply:
done